src/Apps/Order/Routes/Shipping2/Shipping2.jest.tsx

```diff
@@ -198,13 +200,17 @@ const verifyAddressWithSuggestions = async (relayEnv, input, suggested) => {
     ]
   }
 
+  const suggestedAddress = suggested
+    ? { ...recommendedAddress, ...suggested }
+    : recommendedAddress
+
   const verificationResult = {
     __typename: "VerifyAddressType",
     verificationStatus: "VERIFIED_WITH_CHANGES",
     suggestedAddresses: [
       {
-        lines: addressLines(recommendedAddress),
-        address: recommendedAddress,
+        lines: addressLines(suggestedAddress),
+        address: suggestedAddress,
```

Comment:
possibly unnecessary optimization - the result could always just be input address + line1: "suggested-change"